engine/tests/phpunit/Elgg/Tests/EntityMocks.php

```diff
@@ -62,6 +63,16 @@ public function get($guid, $type = '') {
 	}
 
 	/**
+	 * Return callback for mocking \Elgg\Database\EntityTable
+	 *
+	 * @param int    $guid GUID of the mock entity
+	 * @return boolean
+	 */
+	public function exists($guid) {
+		return $guid && array_key_exists($this->mocks[$guid]);
```

Comment:
Just noticed this. I have only included this because it was calling `elgg_entity_exists()`, but I removed after a refactor.